cmake/custom/eigen.cmake

```diff
@@ -1 +1,2 @@
-find_package(Eigen3 REQUIRED)
+find_package(Eigen3 3.1.0 REQUIRED)
```

Comment:
You could use the [imported target](http://eigen.tuxfamily.org/dox/TopicCMakeGuide.html) for Eigen (you'll need to upgrade to 3.3.0 though). And then instead of having the global `include_directories` here you can have `target_link_libraries(example Eigen3::Eigen)` wherever needed.

Reply:
I will incorporate this with https://github.com/MRChemSoft/mrchem/issues/79, currently we cannot use Eigen3.3 on stallo and travis